### DISCUSSION TITLE
Default exit handler for all workflows on controller

### DISCUSSION THREAD

**Discussion by sundbing** [argoproj/argo-workflows#5706] [Q&A] [answered]
As we are running workflows automatically, based on workflowtemplates, created by individuals and automation,
we need to make sure, that each workflow is ending with an exit handler that is defined global.

This exit handler should run in all cases, if defined or not. 

How to achive this? Any idea?

**Comment by terrytangyuan** [comment, marked as answer]
You should be able to set your default exit handler and possibly other default values in `workflowDefaults` of your controller's configmap. See instructions here: https://argoproj.github.io/argo-workflows/default-workflow-specs/
